packages/main/src/Carousel.js

```diff
@@ -110,6 +111,18 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Defines the style of the page indicator.
+		 * @type {sap.ui.webc.main.types.CarouselPageIndicatorStyle}
+		 * @since 1.10
```

Comment:
Add list with current CarouselPageIndicatorStyle enum values since enums are not visible in current playground.